### PR TITLE
add integration tests

### PR DESCRIPTION
**Pull request by SlavoKrupa**
I created integration tests for booster and updated the message content in name service.
@cescoffier Can you PTAL?

### CODE REVIEW

**Review by cescoffier — APPROVED** [review]

File: greeting-service/src/main/java/io/openshift/booster/GreetingServiceVerticle.java
Comment (Ladicek) [inline greeting-service/src/main/java/io/openshift/booster/GreetingServiceVerticle.java]: `"OK"` doesn't look like JSON to me.

**Comment by Ladicek** [comment]
It doesn't really matter, but the Spring Boot and Swarm boosters call the directory just `tests`.

**Comment by cescoffier** [comment]
I prefer calling it integration-tests.

**Comment by cescoffier** [comment]
> On 13 Jun 2017, at 10:13, Ladislav Thon <notifications@github.com> wrote:
> 
> @Ladicek commented on this pull request.
> 
> In greeting-service/src/main/java/io/openshift/booster/GreetingServiceVerticle.java <https://github.com/openshiftio-vertx-boosters/vertx-circuit-breaker-booster/pull/1#discussion_r121609795>:
> 
> > @@ -39,12 +40,12 @@ public void start() throws Exception {
>  
>          Router router = Router.router(vertx);
>  
> -        router.get("/health").handler(rc -> rc.response().end("OK"));
> +        router.get("/health").handler(rc -> rc.response().putHeader(CONTENT_TYPE.toString(), APPLICATION_JSON.toString()).end("OK"));
> "OK" doesn't look like JSON to me.
> 
Why should /health return JSON outcome. Only the status code matters for OpenShift.

**Comment by SlavoKrupa** [comment]
Because header mentions content type and the content doesn't fulfill this contract (I removed the header) as it should not be necessary for OS. 
BTW I do not have write access to this repository. Can you please @cescoffier set up webhooks for testing  and merge this if you are happy with current state ?

**Comment by Ladicek** [comment]
I actually removed my comment about invalid JSON because a string literal _is_ a valid JSON, though typically JSON endpoints return either an object or an array. But I don't really mind :-)

EDIT: ah, ok, I was right in the comment that I deleted and I'm wrong in this comment. Per RFC 4627, "A JSON text is a serialized object or array." But this PR is good now, so nevermind.

**Pull request merged**
